onnx/shape_inference/implementation.h

```diff
@@ -58,6 +74,7 @@ struct InferenceContextImpl : public InferenceContext {
     }
     return &allOutputTypes_[index];
   }
+  std::vector<const TensorProto*> allInputInitializers_;
```

Comment:
name it as "all_inputs" or "all_input_tensors" may be better since you're putting nullptr for an input if it's not an initializer.

Reply:
I think allInputData_ should keep it consistent